<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDlg</class>
 <widget class="QWidget" name="MainDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QWidget" name="widget_6" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_left">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#frame_left{
	background-color: rgb(46, 46, 46);
}
#rbut_message::indicator{
	width:0px;
	height:0px;
}
#rbut_message{
	background:transparent;
	border-image: url(:/imageSource/chat_default.png);
}
#rbut_message:hover{
	background:transparent;
	border-image: url(:/imageSource/chat_hover.png);
}
#rbut_message:checked{
	background:transparent;
	border-image: url(:/imageSource/chat_checked.png);
}
#rbut_contacts::indicator{
	width:0px;
	height:0px;
}
#rbut_contacts{
	background:transparent;
	border-image: url(:/imageSource/lxr_default.png);
}
#rbut_contacts:hover{
	background:transparent;
	border-image: url(:/imageSource/lxr_hover.png);
}
#rbut_contacts:checked{
	background:transparent;
	border-image: url(:/imageSource/lxr_checked.png);
}
#tbut_other{
	background:transparent;
	border:none;
	border-image: url(:/imageSource/other_default.png);
}
#tbut_other:hover{
	background:transparent;
	border:none;
	border-image: url(:/imageSource/other_hover.png);
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>30</number>
         </property>
         <item>
          <widget class="QLabel" name="label_head">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbut_message">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbut_contacts">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="tbut_other">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <property name="minimumSize">
         <size>
          <width>283</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>283</width>
          <height>999999</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#widget_5{
	border:2px solid rgb(214, 214, 214);
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_chatlist" native="true">
           <property name="styleSheet">
            <string notr="true">#widget_chatlist{
	background-color: rgb(247, 247, 247);
	border-right:2px solid rgb(214, 214, 214);
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#widget_2{
	background-color: rgb(245, 245, 245);
}
QToolButton{
	background:transparent;
	border:none;
}
QToolButton:hover{
	background:transparent;
	border:none;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 0, 0, 200), stop:1 rgba(255, 255, 255, 255));
}

QToolButton#tbut_normal:hover{
	background:transparent;
	border:none;
	background-color: rgb(226, 226, 226);
}
QToolButton#tbut_mini:hover{
	background:transparent;
	border:none;
	background-color: rgb(226, 226, 226);
}</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="tbut_mini">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>40</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tbut_normal">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>40</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>□</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tbut_close">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>40</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>×</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_chat" native="true">
           <property name="styleSheet">
            <string notr="true">#widget_chat{
	background-color: rgb(245, 245, 245);
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
